sktime/transformations/hierarchical/reconcile.py

```diff
@@ -94,13 +99,14 @@ def _fit(self, X, y=None):
         if X.index.nlevels < 2:
             return self
 
-        # check index and bottom level of hierarchy are named correctly
-        #   if not, add totals to X
-        if not _check_index_good(X):
+        # check index for no "__total", if not add totals to X
+        if _check_index_no_total(X):
```

Comment:
I don't think this warning is sensible for `_fit`, since nothing is returned. The user would be confused about where things are added.